test/1_stdlib/Collection.swift

```diff
@@ -1,6 +1,13 @@
-// RUN: %target-run-simple-swift | FileCheck %s
+// RUN: %target-run-simple-swift --stdlib-unittest-in-process | tee %t.txt
+// RUN: FileCheck %s < %t.txt
+// note: remove the --stdlib-unittest-in-process once all the FileCheck tests
+// have been converted to StdlibUnittest
```

Comment:
`--stdlib-unittest-in-process` is done because otherwise it repeats all of the non-`StdlibUnittest` tests in the child process, which causes duplicate output.